UPDATING.md

```diff
@@ -24,6 +24,24 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+### Some task context variables are removed
+The following task context variables are removed. Some are obsolete, some are duplicates whose 
+value is also given via other variables, and some are "derived" values from other variables which 
+are also given in the task context.
+
+* `yesterday_ds`: can be derived from `execution_date`
```

Comment:
Can you also show the derivation here?

Reply:
Sure, I've added this example to [the docs](https://github.com/apache/airflow/pull/5010/files#diff-35b9954899f172a9d4e1b8488cade790R85), and added examples for all removed variables to [UPDATING.md](https://github.com/apache/airflow/pull/5010/files#diff-bb7ee25332d5fdca3ead2f545d72b359R32).